subprojects/core/src/test/groovy/org/gradle/api/internal/project/taskfactory/AnnotationProcessingTasks.java

```diff
@@ -456,10 +456,16 @@ public File getOutputFile() {
     }
 
     public static class TaskWithOptionalNestedBean extends TaskWithAction {
+        private final Bean bean;
+
+        public TaskWithOptionalNestedBean(Bean bean) {
```

Comment:
There's somewhere where we now call `new TaskWithOptionalNestedBean()` that should be converted to `new TaskWithOptionalNestedBean(null)`

Reply:
I re-added the no-args constructor.